packages/ag-charts-community/src/chart/series/seriesLayerManager.ts

```diff
@@ -150,12 +180,13 @@ export class SeriesLayerManager {
     }
 
     public destroy() {
-        for (const groups of Object.values(this.groups)) {
-            for (const groupInfo of Object.values(groups)) {
+        this.groups.forEach((groups) => {
+            groups.forEach((groupInfo) => {
                 this.seriesRoot.removeChild(groupInfo.group);
-            }
-        }
-        (this as any).groups = {};
-        (this as any).series = {};
+            });
+        });
+
+        (this as any).groups = new Map();
+        (this as any).series = new Map();
```

Comment:
```suggestion
        this.groups.clear();
        this.series.clear();
```
not: cleaner, and respects being `readonly`